packages/rule-https-only/README.md

```diff
@@ -0,0 +1,85 @@
+# https-only (`https-only`)
+
+`https-only` check if your site is using HTTPS and warns against having
+mixed content.
+
+## Why is this important?
+
+HTTPS is important to guarantees content integrity. Even when your site
+doesn't have sensitive information, an attacker can change the content
+or inject scripts.
+
+## How to use this rule?
+
+To use it you will have to install it via `npm`:
+
+```bash
+npm install @sonarwhal/rule-https-only
+```
+
+Note: You can make `npm` install it as a `devDependency` using the `--save-dev`
+parameter, or to install it globally, you can use the `-g` parameter. For
+other options see
+[`npm`'s documentation](https://docs.npmjs.com/cli/install).
+
+And then activate it via the [`.sonarwhalrc`][sonarwhalrc]
+configuration file:
+
+```json
+{
+    "connector": {...},
+    "formatters": [...],
+    "parsers": [...],
+    "rules": {
+        "https-only": "error"
+    },
+    ...
+}
+```
+
+## What does the rule check?
+
+This rule checks two things:
+
+* The url is served using HTTPS
+* If so, all the content served is HTTPS too.
+
+### Examples that **trigger** the rule
+
+If your site is not served as HTTPS.
+
+```
+sonarwhal http://example.com
+```
+
+If your site is served as HTTPS, but your assets are HTTP.
+
+```html
+<body>
+    <img src="http://assets.com/image.png" />
+    <script src="http://assets.com/script.js"></script>
+</body>
+```
+
+### Examples that **pass** the rule
+
+Your site is served as HTTPS and your assets too.
+
+```html
+<body>
+    <img src="https://assets.com/image.png" />
+    <script src="https://assets.com/script.js"></script>
+</body>
+```
+
+## Can the rule be configured?
+
+This rule cannot be configured.
```

Comment:
I'd remove this section. Don't think we have it for rules that can't be configured.